{"uuid":"d08cfeec-af06-4085-9c10-30010ee97590","historyId":"11b57b3d11f0da5f8a3abeba3d5bb920","fullName":"dsAlgoTests.ArrayPageTest.submitInvalidCodeSqrsOfSortedArrayQtn","labels":[{"name":"package","value":"dsAlgoTests.ArrayPageTest"},{"name":"testClass","value":"dsAlgoTests.ArrayPageTest"},{"name":"testMethod","value":"submitInvalidCodeSqrsOfSortedArrayQtn"},{"name":"parentSuite","value":"DSAlgoCrossBrowser"},{"name":"suite","value":"FireFoxTest"},{"name":"subSuite","value":"dsAlgoTests.ArrayPageTest"},{"name":"host","value":"KAITLYN"},{"name":"thread","value":"30928@KAITLYN.TestNG-tests-2(39)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"submitInvalidCodeSqrsOfSortedArrayQtn","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [Error occurred during submission] but found [ No tests were collected]","trace":"java.lang.AssertionError: expected [Error occurred during submission] but found [ No tests were collected]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1590)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:150)\r\n\tat org.testng.Assert.assertEquals(Assert.java:132)\r\n\tat org.testng.Assert.assertEquals(Assert.java:656)\r\n\tat org.testng.Assert.assertEquals(Assert.java:666)\r\n\tat dsAlgoTests.ArrayPageTest.submitInvalidCodeSqrsOfSortedArrayQtn(ArrayPageTest.java:321)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\r\n\tat org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\r\n\tat org.testng.TestRunner.run(TestRunner.java:616)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)\r\n\tat org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)\r\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\r\n\tat java.base/java.lang.Thread.run(Thread.java:1583)\r\n"},"stage":"finished","steps":[],"attachments":[{"name":"Screenshot on Failure","source":"630704c6-9ade-4d32-ad1f-b3b997320780-attachment.png","type":"image/png"}],"parameters":[{"name":"arg1","value":"Error occurred during submission"},{"name":"browser","value":"firefox"},{"name":"arg0","value":"print(\"Hello, World!\""}],"start":1748070126934,"stop":1748070129453}